### ISSUE TITLE
queryExchangeState return value is String and not LinkedTreeMap

### ISSUE THREAD

**Issue by marinatraisler** [CCob/bittrex4j#23]

**Comment by marinatraisler** [comment]
and also you forgot to add decode to the queries

**Comment by CCob** [comment]
Yup, looks like I missed this call as part of the upgrade to the c2 hub, thanks for letting me know.  I've just published 1.0.10-SNAPSHOT that should solve this issue.  Let me know if this works for you and I can get it merged and cut a new release.

**Comment by marinatraisler** [comment]
yeap its working for me
